pkgs/games/linthesia/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, python3, libGL, libGLU
+, alsa-lib, glibmm, sqlite, SDL2, SDL2_ttf, SDL2_image, gtk3, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "linthesia";
+  version = "unstable-2022-01-31";
+
+  src = fetchFromGitHub {
+    owner = "linthesia";
+    repo = "linthesia";
+    rev = "11669e355c72fefefb2b3adee31e821dc4c4c9ce";
+    sha256 = "sha256-FmzQjd9rR9NS8za/kgdCACaEqk2J5KpYrjInC0q9Ep4=";
+  };
+
+  postPatch = ''
+    patchShebangs meson_post_install.py
+  '';
+
+  nativeBuildInputs = [ meson ninja pkg-config python3 wrapGAppsHook ];
+  buildInputs = [
+    libGL
+    libGLU
+    alsa-lib
+    glibmm
+    sqlite
+    SDL2
+    SDL2_ttf
+    SDL2_image
+    gtk3.out # icon cache
+  ];
+
+  meta = with lib; {
+    description = "A game of playing music using a MIDI keyboard following a MIDI file";
+    homepage = "http://linthesia.sourceforge.net";
```

Comment:
Nit: is this homepage really a good choice?  By looking at it I'd think the SW has been abandoned for 10+ years.  I'd suggest `inherit (src.meta) homepage;` instead.